pages/sigmaker.tsx

```diff
@@ -42,37 +51,119 @@ const Sigmaker: NextPage = () => {
     />
   );
 
+  const createSignature = () => {
+    if (signatureData) {
+      return (
+        <div>
+          <Typography variant="h4">
+            Paste this into Gmail!
```

Comment:
After adding some line-height to the text (see other comment), this header will be pretty close to the top of the signature created. Let's add some bottom padding to 'Paste this into Gmail!'